selfdrive/ui/translations/main_pt-BR.ts

```diff
@@ -1199,12 +1199,12 @@ trabalho definido</translation>
     <message>
         <location filename="../qt/offroad/settings.cc" line="58"/>
         <source>🌮 End-to-end longitudinal (extremely alpha) 🌮</source>
-        <translation type="unfinished"></translation>
+        <translation>🌮 End-to-end longitudinal (experimental) 🌮</translation>
```

Comment:
no translation?

Reply:
A more explicit translation would be too long and perhaps lose the semiotics. Ex: "solução de ponta a ponta" means "end-to-end". All the others worlds are the same on portuguese from Brazil.